ts/Core/Series/Point.ts

```diff
@@ -417,7 +417,7 @@ class Point {
 
             for (const prop in point) { // eslint-disable-line guard-for-in
```

Comment:
`for in` is not sufficient for classes anymore, because it does not catch inherited functions.